jaystaks/Politico
# Panelist/LFA feedback implementations

### What does this PR do?
Implement panelist and lfa feedback on challenge 3
### Description of Task to be completed?

1. Create a user and save on a DB
2.  Login a user and receive a token for the session
3.  Added validations when signing up with non-Kenyan phone and non-email characters.
4.  Register for office and party
### How should this be manually tested?
Go to github and clone repo https://github.com/jaystaks/Politico, switch to develop branch

  -  Make sure to have the python _virtualenv_ if not `pip install virtualenv`
  -  Create the env with this command `virtualenv venv`
  -   For linux type this on the terminal to access your virtual environment `. venv/bin/activate`
  -  Install the packages from the requirements.txt by using this commands `pip install -r requirements.txt`
  - Start your postgresql server with the following command `sudo systemctl start postgresql` 
  - Run `flask init-db` to create the database tables (schemas)
  -  And finally run` python run`

To test the endpoints, open POSTMAN send a PATCH, POST and GET requests to the endpoint http://127.0.0.1:5000/api/v2/auth/login for login requests or http://127.0.0.1:5000/api/v2/auth/signup for signingup requests.

What are the relevant pivotal tracker stories?
(https://www.pivotaltracker.com/n/projects/2241696)
